INSTALLING.rst

```diff
@@ -131,21 +128,16 @@ Install prerequisites
          A `temporary bugfix branch of HIP <https://github.com/glotzerlab/HIP/tree/hipfuncgetattributes_revertvectortypes>`_
```

Comment:
The block starting with `A temporary bugfix branch` no longer applies and should be removed. The ROCm requirement should be increased to >= 3.5.0. (#750).